CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Uncategorized
+
+- fix: update corepack enable
+- fix: yarn
+- chore: add build script
+- fix: add allowscript
+- chore: remove node 18.x ci check
+- feat: github ci workflow
+- feat: rename package
+- Merge branch 'main' into feat/rn0.76
+- fix: filterByAuthorizedAccounts
+- Update README.md ([#6](git+https://github.com/MetaMask/react-native-acm/pull/6))
+- fix: require @react-native-community/cli
+- feat: support rn 0.76
+- Feat/mm mobile ([#4](git+https://github.com/MetaMask/react-native-acm/pull/4))
+- feat: google SignIn android works ([#1](git+https://github.com/MetaMask/react-native-acm/pull/1))
+- fix: add yarn lock file
+- feat: google SignIn android works
+- chore: initial commit
+
+[Unreleased]: git+https://github.com/MetaMask/react-native-acm/
```

Comment:
This URL seems to be invalid. The `auto-changelog` tool derives these links from the repository URL in `package.json`, so what are your thoughts on changing that to `https://github.com/MetaMask/react-native-acm.git`?

Reply:
I tried to change it to `https://github.com/MetaMask/react-native-acm.git` but getting error here
https://github.com/MetaMask/react-native-acm/actions/runs/15967924226/job/45032333443?pr=8

Reply:
What if you were to take off the `git+` from the URL?

```
  "repository": {
    "type": "git",
    "url": "https://github.com/MetaMask/react-native-acm.git"
  },
```